docs/source/tutorials/instrument-modeling/bond-lifecycling.rst

```diff
@@ -95,9 +95,9 @@ The investor then claims the effect:
 .. code-block:: daml
 
   -- Claim effect
-  lifecycleClaimRuleCid <- toInterfaceContractId @Claim.I <$> submitMulti [custodian, investor] [] do
-    createCmd Claim.Rule
-      with
+  lifecycleClaimRuleCid <- toInterfaceContractId @Claim.I <$>
+    submitMulti [custodian, investor] [] do
```

Comment:
We should not need this `do` as it's a single statement. A `$` should suffice.

Reply:
But I think we use `do` consistently throughout, no? I'd rather not introduce mixed style here, as it'll be confusing

Reply:
maybe easier for a new person to the language to understand `submit bob do ...` than `submit bob $ `